README.md

```diff
@@ -29,20 +29,111 @@ The <b>Agglayer</b> (<i>Aggregation layer</i>) provides a common language for se
+|--------------------------------|--------------------------------------------------------------------------------------------------------------|
+| agglayer-aggregator-notifier   | Contains certificate implementations for `certify` and `settled`, as well as `packer`, which handles the packing of an epoch |
+| agglayer-certificate-orchestrator | Manages the orchestration and handling of certificates; also handles `current_epoch`, which allows non-orchestrators to push a proven certificate |
+| agglayer-clock                 | Defines the pace of the Agglayer in terms of epoch with support for two clocks: time (for testing) and block (for listening for L1 blocks) |
+| agglayer-config                | Manages configuration settings and parameters for Agglayer components                                       |
+| agglayer-contracts             | Contains smart contracts and related logic                                                                  |
+| agglayer-gcp-kms               | Provides integration with GCP's Key Management Service for secure key handling                              |
+| agglayer-node                  | Responsible for spawning and running the different components of the node                                   |
+| agglayer-prover-types          | Defines data structures and types used by the prover                                                        |
+| agglayer-prover                | Responsible for running everything related to the prover                                                    |
+| agglayer-signer                | Manages signing operations                                                                                  |
+| agglayer-storage               | Contains two layers: a physical layer for abstracting RocksDB and a logic layer for exposing the interface to other crates so that they may interact with the storage |
+| agglayer-telemetry             | Handles telemetry and monitoring functionalities                                                            |
+| agglayer-types                 | Defines common data types and structures                                                                    |
+| agglayer                       | The CLI for interacting with the Agglayer                                                                   |
+| pessimistic-proof-program      | Implements the pessimistic proof program                                                                    |
+| pessimistic-proof-test-suite   | Provides a suite of tests for validating the functionality of the pessimistic proof program                  |
+| pessimistic-proof              | Contains the core logic and implementation of the pessimistic proof mechanism                               |
+
+## Prerequisite
+
+Before working with the repository, you’ll need the following:
+
+### Succinct Prover Network
```

Comment:
The goal of the "Getting started | Prerequisite" is to cover which user journey?

- Cloning and running tests, in order to learn more and contribute ?
- Building and deploying an instance ?

Reply:
The first option:

* Cloning and running tests, in order to learn more and contribute

For more end-to-end play, we're going to point people to the Kertosis-CDK repository. 

In terms of building and deploying an instance, I'm not sure, tbh. I spoke with Mudit last week and he mentioned eventually we'll have a way for people to do build instances from the AggLayer repo but at this stage this is more about introducing people to the code that's in this repo.

Reply:
I would say that if the goal is to engage people to contribute, I would place all information related to how to install, run and test into a `CONTRIBUTING.md` which is the standard point in open-source to define the different actions and standard for user to contribute. This file will be accessible from the top right menu, and can be listed in the README. If we feel that the `CONTRIBUTING` file is getting too big, or that we should keep it standard, we could have a `docs` folder containing this specific setups.

For running, I'm expecting to have more docs that the README, maybe file like terraform files to configure and run the different binary.

More generally, I would prefer a small and readable README, that define what's going on, what's the purpose of the code in this repo and links to resources to either, learn more about the concept or deep dive into the implementation. As an example, I like the shape of [SUI](https://github.com/MystenLabs/sui/blob/main/README.md) README or Aptos [one](https://github.com/aptos-labs/aptos-core/blob/main/README.md).

Reply:
Interesting approach @Freyskeyd . Pinging @pgebheim for thoughts, as the structure was developed in coordination with he and Jaime.

Reply:
Hey @Freyskeyd -- 

I love the vision here. For simplicity and to get it moving, what if we just split out the CONTRIBUTING stuff right now (I can do it), and then we can merge and start iterating as we get more of the docs together and deployed. E.g. as "running" gets into the docs folder we can just link there.

Right now the v0.2 docs (for IPs, integrators) are in process of being put into the main docs site etc.

Would that work for you?

Reply:
Actually @Freyskeyd -- CONTRIBUTING.md is supposed to be for contribution guidelines, style guides etc. For example see what you linked to: https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md

My main goal with this is to get something moving so I think we should merge this as is and then iterate.

Reply:
Resolving this for now and we can iterate when the more complete docs are avaialble.